index.js

```diff
@@ -83,7 +130,25 @@ function countByRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+
+  let movName = null;
+
+  if(movies.length < 1){
+    return movName;
+  }
+
+  for(let i = 0; i < movies.length; i++){
+
```

Comment:
We have an issue here with our else statement.  Walk through what would happen if the _first movie did not match the id_.